obs_text_source.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <chrono>
+#include <string>
+
+// Add `_path` as a text source in obs to get an onscreen queso-queue timer.
+class ObsTextSource {
+  public:
+    void UpdateTime(std::chrono::seconds seconds);
+
+  private:
+    void WriteMessage(std::string message);
+    std::string _currentMessage;
+    std::string _path = "textsource.txt";
```

Comment:
Probably better to take this at the constructor so we can use the same trick if we want something else (for example, who's next in queue)

Reply:
Done.